nixos/common/networking.nix

```diff
@@ -24,4 +24,14 @@
   systemd.services.systemd-networkd.stopIfChanged = false;
   # Services that are only restarted might be not able to resolve when resolved is stopped before
   systemd.services.systemd-resolved.stopIfChanged = false;
+
+  # Allows to find machines on the local network by name
+  systemd.network.networks."99-ethernet-default-dhcp".networkConfig = {
+    MulticastDNS = lib.mkDefault "yes";
+    LLMNR = lib.mkDefault "yes";
```

Comment:
LLMNR is being deprecated and even Microsoft recommends disabling it.

Reply:
I didn't know that Windows does now mdns as well.